### PR TITLE
add @jengeb, @basiclaser and @bitgal to team page

### PR DESCRIPTION
**Pull request by andrenarchy**

### CODE REVIEW

File: html/about.html [hunk 1]
```diff
@@ -162,6 +162,93 @@ <h3 class="ph-no-margin-top ph-heading-light">
...
         <div class="col-xs-4">
+          <img class="img-circle img-responsive" src="./static/img/jenny.jpg" alt="Jennifer Gebske" />
+        </div>
+        <div class="col-xs-8">
+          <h3 class="ph-no-margin-top ph-heading-light">
+            Jennifer Gebske
+            <small class="ph-newline">Full-stack developer</small>
+          </h3>
+          <p>
+            Studied theatre and media studies, economics and computer science. Loves food, travelling and volleyball.
```
Comment (nschloe) [inline html/about.html]: Oxford comma, but never mind now.

File: html/about.html [hunk 2]
```diff
@@ -162,6 +162,93 @@ <h3 class="ph-no-margin-top ph-heading-light">
...
+        <div class="col-xs-4">
+          <img class="img-circle img-responsive" src="./static/img/christopher.jpg" alt="Christopher Shelley" />
+        </div>
+        <div class="col-xs-8">
+          <h3 class="ph-no-margin-top ph-heading-light">
+            Christopher Shelley
+            <small class="ph-newline">Backend developer</small>
+          </h3>
+          <p>
+            Javascript developer, loves making, mixing and breaking tools. Enjoys living in different places.
```
Comment (nschloe) [inline html/about.html]: Ditto.

File: html/about.html [hunk 3]
```diff
@@ -162,6 +162,93 @@ <h3 class="ph-no-margin-top ph-heading-light">
...
+        <div class="col-xs-4">
+          <img class="img-circle img-responsive" src="./static/img/osnat.jpg" alt="Osnat Vider" />
+        </div>
+        <div class="col-xs-8">
+          <h3 class="ph-no-margin-top ph-heading-light">
+            Osnat Vider
+            <small class="ph-newline">Backend developer</small>
+          </h3>
+          <p>
+            In Berlin since 2011. Enjoys Ruby, her family and the next vacation in Tel-Aviv.
```
Comment (nschloe) [inline html/about.html]: Ditto. ... Perhaps this is worth another commit now.

Comment (andrenarchy) [inline html/about.html]: LOL! fixed.

**Pull request merged**